### PR TITLE
Feature ➤ Aspect ratio

### PR DESCRIPTION
**Pull request by EAlexisDuque**

### CODE REVIEW

File: app/src/main/java/co/condorlabs/customcomponents/imagecropview/AppCompatCropImageView.java [hunk 1]
```diff
@@ -142,6 +140,12 @@ protected void onDraw(Canvas canvas) {
                 resizeDrawable2.draw(canvas);
                 resizeDrawable3.draw(canvas);
             }
+            // Calculate initial aspectRatio
```
Comment (leidyzulu) [inline app/src/main/java/co/condorlabs/customcomponents/imagecropview/AppCompatCropImageView.java]: should create a method called calculateInitialAspectRatio

File: app/src/main/java/co/condorlabs/customcomponents/imagecropview/AppCompatCropImageView.java [hunk 2]
```diff
@@ -250,6 +287,8 @@ private void init(@Nullable AttributeSet attrs) {
         resizeDrawable1.setTint(cornerColor);
         resizeDrawable2.setTint(cornerColor);
         resizeDrawable3.setTint(cornerColor);
+        // keep aspect ratio
```
Comment (leidyzulu) [inline app/src/main/java/co/condorlabs/customcomponents/imagecropview/AppCompatCropImageView.java]: the variable says the function, Is it necessary the comment?

File: app/src/main/java/co/condorlabs/customcomponents/imagecropview/AppCompatCropImageView.java (hunk 1 above)
Comment (EAlexisDuque) [reply app/src/main/java/co/condorlabs/customcomponents/imagecropview/AppCompatCropImageView.java]: Ok

File: app/src/main/java/co/condorlabs/customcomponents/imagecropview/AppCompatCropImageView.java (hunk 2 above)
Comment (EAlexisDuque) [reply app/src/main/java/co/condorlabs/customcomponents/imagecropview/AppCompatCropImageView.java]: Ok

**Pull request merged**